PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="wrapper fadeInDown" >
+    <p class="card-subtitle mb-2 text-muted"> {{ product.category }} </p>
+
+    <p class="name" >{{product.name}}</p>
+    
+    
+    
+    <ngb-carousel wrap="true" style="width:30%;margin-left: 5%;float: left;" [interval]="false" *ngIf="product.imagesUrl.length > 1; else noCarousel" #carousel>
```

Comment:
i think you should move style to .scss file

Reply:
@Suchar230 please check it

Reply:
Html styling when scss already in use risks unexpected behaviour and makes code unmaintainable